sycl/test-e2e/KernelFusion/Reduction/reduction.cpp

```diff
@@ -1,5 +1,6 @@
 // RUN: %{build} -fsycl-embed-ir -o %t.out
 // RUN: %{run} %t.out
+// XFAIL: cpu
```

Comment:
Interesting... If pre-commit passes, that would mean that the test is skipped there.

Reply:
If this is caused by a race condition depending on the strategy being used for the reduction, we may only run into the error in some devices